frontend/src/components/cards/counties-map/counties-map.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactEcharts from 'echarts-for-react';
+import { Card } from '../../layout/card/card';
+import { mnemonics } from '../../../config/mnemonics';
+import { Constants } from '../../../config/globals';
+
+export const EMBED_COUNTIES_MAP = 'counties-map';
+
+export class CountiesMap extends React.PureComponent {
+  getChartOptions() {
+    const data = this.props.state.counties.map(countie => {
+      return {
+        name: mnemonics[countie.name],
+        value: countie.value
+      };
+    });
+    return {
+      tooltip: {
+        trigger: 'item'
+      },
+      visualMap: {
+        show: false,
+        min: 0,
+        //max: this.props.state.max,
+        max: 0,
+        left: 'left',
+        top: 'bottom',
+        text: ['Ridicat', 'Scazut'],
+        calculable: true,
+        inRange: {
+          color: [Constants.countyLowestColor, Constants.countyHighestColor]
+        }
+      },
+      series: [
+        {
+          name: 'Cazuri',
+          type: 'map',
+          mapType: 'RO',
+          itemStyle: {
+            areaColor: Constants.curedColor
+          },
+          label: {
+            normal: {
+              show: false
+            }
+          },
```

Comment:
Replacing this section with the following code will fix #286 by hiding all the county names from the map
```javascript
emphasis: {
    label: {
        show: false
    }
},
```